src/Client/LiveStripeClient.php

```diff
@@ -44,8 +42,21 @@ public function createPaymentIntent(array $createPayload): PaymentIntent
         return $this->stripeClient->paymentIntents->create($createPayload);
     }
 
-    public function retrievePaymentMethod(string $paymentMethodId): PaymentMethod
+    public function updatePaymentMethodBillingDetail(string $paymentMethodId, Donation $donation): PaymentMethod
     {
-        return $this->stripeClient->paymentMethods->retrieve($paymentMethodId);
+        // "A PaymentMethod must be attached a customer to be updated."
+        $this->stripeClient->paymentMethods->attach($paymentMethodId, ['customer' => $donation->getPspCustomerId()]);
```

Comment:
Have you confirmed that it's OK to attach the same method to the same customer multiple times? Since the function name doesn't indicated that it will attach the method we shouldn't expect the caller to only call it once.

Reply:
I just tried it in test mode with `pm_card_visa` – seems to be allowed and succeed. Hopefully real payment methods will be the same.